content/v2.0/cloud/get-started.md

```diff
@@ -18,29 +18,34 @@ _See the differences between {{< cloud-name "short">}} and InfluxDB OSS
 [below](#differences-between-influxdb-cloud-and-influxdb-oss)._
 
 ## Start for free
+
 Start using {{< cloud-name >}} at no cost with the [Free Plan](/v2.0/cloud/pricing-plans/#free-plan).
 Use it as much and as long as you like within the plan's rate-limits.
 Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortably.
 Once you're ready to grow, [upgrade to the Usage-based Plan](/v2.0/cloud/account-management/upgrade-to-usage-based-plan/).
 
 ## Sign up
 
-1. Go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your email address and password, and click **Sign Up**.
-2. InfluxDB Cloud requires email verification to complete the sign up process. Verify your email address by opening
-   the email sent to the address you provided and clicking **Verify Your Email**.
-3. Choose your cloud provider. Currently, **AWS** is the only cloud provider available.
+1. Choose one of the following:
+   - To subscribe for a plan through InfluxData, go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your credentials, and then click **Sign Up**.
```

Comment:
Thoughts in general about terminology?
We use “plan” and AWS uses “subscription.” Does it make sense to use the same term consistently for clarity (or keep more descriptive re context)?

Reply:
I know there's already been some bikeshedding on this with the marketing team and "subscription" was one of the proposals. Before InfluxDB Cloud 2.0 launched, we decided to go with "Free Plan" and "Pay As You Go Plan" (now "Usage-Based Plan"). I think it's more important to align with the language of the marketing team than with AWS's terminology, but that's only my opinion.

Reply:
I agree with @sanderson -- let's use Marketing terminology.

I do think there's a semantic difference -- we expose a Free Plan and a Usage-Based Plan. And you can subscribe to those plans through either InfluxData (Organic) or AWS (Marketplace).

Reply:
@lukevmorris, where will they be able to select Free plan vs Usage-Based Plan in AWS? After hitting the "Set Up Your Account" button?
...........................
- To subscribe to a **Free Plan** or **Usage-Based Plan** through InfluxData/AWS Marketplace
OR
- To subscribe to an **InfluxDB Cloud 2.0 plan** through InfluxData/AWS Marketplace...

Reply:
@kelseiv They won't have a selection -- clicking Subscribe in AWS locks you into a Usage-Based plan.

We may at some point in the future allow users to begin with a free trial or free version. But not on launch.